### ISSUE TITLE
More cases of spaced-comment

### ISSUE THREAD

**Issue by johnd0e** [eslint/eslint#11287]
**What rule do you want to change?** spaced-comment

**Does this change cause the rule to produce more or fewer warnings?** fewer

**How will the change be implemented? (New option, new default behavior, etc.)?** option

**Please provide some example code that this change will affect:**

```js
// spaced comment
/////// also spaced comment ///////
//=== and one more ===//
```

**What does the rule currently do for this code?** only 1st line is correct

**What will the rule do after it's changed?** rest 2 lines also valid as there is space before actual text

**Are you willing to submit a pull request to implement this change?** no

**Comment by kaicataldo** [comment]
Thanks for the issue. Do you mind clarifying how your proposal differs from the current [markers](https://eslint.org/docs/rules/spaced-comment#markers) configuration option?

**Comment by johnd0e** [comment]
Marker length is fixed.
E.g. to make second sample correct I could set `{ "markers": ["/////"] }`.
But to make the rule more general I have to define all possible strings, `["/","//","///","////","/////", ...<infinite>].`

**Comment by kaicataldo** [comment]
That's correct. Can you update this proposal to outline what the solution you're proposing is (filling out the `How will the change be implemented?` section with a detailed proposal will help us out here).

**Comment by johnd0e** [comment]
> Can you update this proposal to outline what the solution you're proposing is

Sure I will, after we discuss some solutions and their possible drawbacks.
Here for start:
* Some option to allow markers to be repeated.
  * E.g. it could be in form of regexp: `{ "markers-re": ["/+", "=+"] }`
* Or we can think about this repeated pattern in terms of `exceptions`.
  * E.g. we can add new options to allow exceptions be inline: `{ "exceptions": ["/", "="],  "exceptions-inline": true}`

**Comment by kaicataldo** [comment]
Sure, happy to discuss. Before we talk about solutions, do you mind clarifying the problem you are trying to solve? Is it just not having to list all possible permutations of `markers`?
